### DISCUSSION TITLE
关于桌面歌词

### DISCUSSION THREAD

**Discussion by IXYRI** [qier222/YesPlayMusic#1481] [Q&A] [answered]
有没有实现桌面歌词的打算？
就是那种一个在所有窗口顶部的、有翻译的歌词字幕。
因为听外文歌的时候比较多，然后又喜欢边看歌词翻译边做事。
这对我真的比较重要，然后因为使用的是Archlinux，Web功能不全（甚至没有收藏的专辑），网易云官方的app很久没有维护了（也没有翻译）

**Comment by pan93412** [comment, marked as answer]
https://github.com/qier222/YesPlayMusic/wiki/YesPlayMusic-使用中可能遇到的问题-(FAQ)#有计划支持-桌面歌词--状态栏歌词吗

**Comment by LiYulin-s** [comment]
我写了一个KDE上的顶栏歌词，感兴趣的话可以试试[url](https://github.com/LiYulin-s/org.kde.plasma.ypm-lyrics)

**Comment by Ghostssswind** [comment]
微软商店的“热词”应用可以匹配 https://apps.microsoft.com/store/detail/%E7%83%AD%E8%AF%8D/9MXFFHVQVBV9?hl=zh-hk&gl=hk
